libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -57,12 +62,14 @@ export class ButtonComponent implements AfterContentInit {
   @Input() set isDestructive(state: boolean) {
     this.destructive = state;
   }
-  @HostBinding('class')
   @Input()
   themeColor: NotificationColor;
   @Input() expand: 'full' | 'block';
   @Input() text: string;
   @Input() isFloating: boolean = false;
+  @HostBinding('class')
+  @Input()
+  size: 'sm' | 'md' | 'lg' = 'md';
```

Comment:
Please align with other component and declare exported enum type for ButtonSize

Reply:
I created an issue for this (https://github.com/kirbydesign/designsystem/issues/1408), so if ok with you, I'll leave it until then to align this for all the comps.

Reply:
Implemented in 5f6a07ab84c1698024f84d3f90825fd39ad84bc6